src/monta_mcp/__init__.py

```
@@ -1,2 +1,314 @@
+        Tuple of (MontaApiClient, aiohttp.ClientSession)
+
+    Raises:
+        ValueError: If credentials are not set in environment variables
+    """
+    client_id = os.getenv("MONTA_CLIENT_ID")
+    client_secret = os.getenv("MONTA_CLIENT_SECRET")
+
+    if not client_id or not client_secret:
+        raise ValueError(
+            "MONTA_CLIENT_ID and MONTA_CLIENT_SECRET environment variables must be set"
+        )
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)